tests/integration/records/serializers/literature/test_schemas_json.py

```diff
@@ -103,7 +103,12 @@ def test_references_schema_with_record(isolated_app):
                             'journal_title': 'PTEP',
                             'journal_volume': '2012',
                             'year': 2012,
-                        }
+                        },
+                        'urls': [
+                            {
+                                'value': 'http://sf2a.eu/semaine-sf2a/2013/proceedings/'
+                            }
+                        ],
```

Comment:
hmm why you need this here?

Reply:
there are test cases missing if it's a linked record and has ``dois`` and ``arxiv_eprints`` to return to correct serialized data